### PR TITLE
Never hide errors

### PR DESCRIPTION
**Pull request by Tbaut**
An issue was reported to me by @freddyli7 but the error must have disappeared before he came back to Multix. As a result, it was hard to know, what went wrong unless you go check subscan (and because it's a multisig extrinsic, this isn't easy either).

I had removed this behavior because I thought it was annoying to users, it turns out that since it's rather an exception, bothering users and asking them to manually close the error isn't that annoying either and may actually help them.

### CODE REVIEW

**Comment by asnaith** [comment]
I attempted a balance transfer of an amount greater than the pure’s balance using a manual extrinsic. 

It failed after threshold approval was met (https://rococo.subscan.io/extrinsic/0xeead1e605c29a02533324f66d65d4c1f4f2a4c6823af7c91ae6ba1686513a3b2) but I never saw anything in the multix UI.

**Comment by Tbaut** [comment]
Oh, that's news to me. Opened a new issue https://github.com/ChainSafe/Multix/issues/198
Batch error are still showing (but not super nicely)

**Comment by asnaith** [comment]
I didn't see an error for a batch tx failure either but I do see it in the console: 

```
16:21:40.368 web3Enable: Enabled 0 extensions: index-fb5980fc.js:249:61774
16:21:40.890 web3Enable: Enabled 1 extension: polkadot-js/0.44.1 index-fb5980fc.js:249:61774
16:21:42.069 No local name to load index-fb5980fc.js:249:67771
16:21:49.001 Transaction status: Ready index-fb5980fc.js:474:857
16:21:49.001 Transaction status: Broadcast index-fb5980fc.js:474:857
16:21:55.023 Transaction status: InBlock index-fb5980fc.js:474:857
16:21:55.024 Included at block hash 0xc4c1a1b91b0da31d4b1aad09d7d3785bb3a5fb79ccead81455f5add7ab1a332a index-fb5980fc.js:474:1027
16:21:55.024 Events: index-fb5980fc.js:474:1086
16:21:55.024 	 {"applyExtrinsic":2} : balances.Withdraw ["5H6rmP3abUDAV7L3JdcboWdbUixgyHG84uzE7ye2VwstfcCZ",76203427] index-fb5980fc.js:474:1186
16:21:55.025 	 {"applyExtrinsic":2} : multisig.MultisigApproval ["5H6rmP3abUDAV7L3JdcboWdbUixgyHG84uzE7ye2VwstfcCZ",{"height":5736191,"index":2},"5Gr8XxcCL96vEmMYYmLNWAQKELHLMRzNiJ1RZ2Cwn3wxwBvc","0x888562dac76ecf60e01e7189728d4e31ff7adfb8ed22a248376c92860eda9a7d"] index-fb5980fc.js:474:1186
16:21:55.025 	 {"applyExtrinsic":2} : balances.Deposit ["5H6rmP3abUDAV7L3JdcboWdbUixgyHG84uzE7ye2VwstfcCZ",39550] index-fb5980fc.js:474:1186
16:21:55.025 	 {"applyExtrinsic":2} : balances.Deposit ["5CtotkWgsDtNZYCshp64gGv7o2tCk2q5nQG1sSYHcYUtMcSC",76163877] index-fb5980fc.js:474:1186
16:21:55.026 	 {"applyExtrinsic":2} : transactionPayment.TransactionFeePaid ["5H6rmP3abUDAV7L3JdcboWdbUixgyHG84uzE7ye2VwstfcCZ",76163877,0] index-fb5980fc.js:474:1186
16:21:55.026 	 {"applyExtrinsic":2} : system.ExtrinsicSuccess [{"weight":{"refTime":243439984,"proofSize":6811},"class":"Normal","paysFee":"Yes"}] index-fb5980fc.js:474:1186
16:22:06.019 Transaction status: Ready index-fb5980fc.js:474:857
16:22:06.019 Transaction status: Broadcast index-fb5980fc.js:474:857
16:22:07.092 Transaction status: InBlock index-fb5980fc.js:474:857
16:22:07.093 Included at block hash 0xb990eaea66cb716cc478510da877fb3c7e12e9a86140f3b191bd38867131911e index-fb5980fc.js:474:1027
16:22:07.093 Events: index-fb5980fc.js:474:1086
16:22:07.093 	 {"applyExtrinsic":2} : balances.Withdraw ["5EcNyAfDsyHgKZej7dZ8Gjs6Zt9PRjc9f6bhijGxXfeB5tcd",109410486] index-fb5980fc.js:474:1186
16:22:07.093 	 {"applyExtrinsic":2} : balances.Unreserved ["5Cqx9dsouTgq48wBuheERrre1fxxdpMktee7RY1T3jZwLNJ4",67279993200] index-fb5980fc.js:474:1186
16:22:07.093 	 {"applyExtrinsic":2} : utility.BatchInterrupted [0,{"token":"FundsUnavailable"}] index-fb5980fc.js:474:1186
16:22:07.093 	 {"applyExtrinsic":2} : proxy.ProxyExecuted [{"ok":null}] index-fb5980fc.js:474:1186
16:22:07.094 	 {"applyExtrinsic":2} : multisig.MultisigExecuted ["5EcNyAfDsyHgKZej7dZ8Gjs6Zt9PRjc9f6bhijGxXfeB5tcd",{"height":5736191,"index":2},"5Gr8XxcCL96vEmMYYmLNWAQKELHLMRzNiJ1RZ2Cwn3wxwBvc","0x888562dac76ecf60e01e7189728d4e31ff7adfb8ed22a248376c92860eda9a7d",{"ok":null}] index-fb5980fc.js:474:1186
16:22:07.094 	 {"applyExtrinsic":2} : balances.Deposit ["5D78Zvg5E59QycySGHemgPsyupbYXVSFaUWFKScYRxTgeNJR",109410486] index-fb5980fc.js:474:1186
16:22:07.094 	 {"applyExtrinsic":2} : transactionPayment.TransactionFeePaid ["5EcNyAfDsyHgKZej7dZ8Gjs6Zt9PRjc9f6bhijGxXfeB5tcd",109410486,0] index-fb5980fc.js:474:1186
16:22:07.095 	 {"applyExtrinsic":2} : system.ExtrinsicSuccess [{"weight":{"refTime":901459335,"proofSize":11517},"class":"Normal","paysFee":"Yes"}] index-fb5980fc.js:474:1186
16:22:11.446 Transaction status: Finalized index-fb5980fc.js:474:857
16:22:11.446 Finalized block hash 0xc4c1a1b91b0da31d4b1aad09d7d3785bb3a5fb79ccead81455f5add7ab1a332a index-fb5980fc.js:474:2170
16:22:23.388 Transaction status: Finalized index-fb5980fc.js:474:857
16:22:23.388 Finalized block hash 0xb990eaea66cb716cc478510da877fb3c7e12e9a86140f3b191bd38867131911e index-fb5980fc.js:474:2170
```

**Comment by asnaith** [comment]
Same when a proposal is sent from the multisig instead of the proxy. The failure appears in the console but not Multix. 

```
16:32:10.366 Transaction status: Ready index-fb5980fc.js:474:857
16:32:10.366 Transaction status: Broadcast index-fb5980fc.js:474:857
16:32:13.015 Transaction status: InBlock index-fb5980fc.js:474:857
16:32:13.015 Included at block hash 0x597356bcc886bb9cb8d732bd845a49cf342cdce463c4a39e8d67319abb27e998 index-fb5980fc.js:474:1027
16:32:13.015 Events: index-fb5980fc.js:474:1086
16:32:13.016 	 {"applyExtrinsic":2} : balances.Withdraw ["5Cqx9dsouTgq48wBuheERrre1fxxdpMktee7RY1T3jZwLNJ4",71587265] index-fb5980fc.js:474:1186
16:32:13.016 	 {"applyExtrinsic":2} : balances.Unreserved ["5H6rmP3abUDAV7L3JdcboWdbUixgyHG84uzE7ye2VwstfcCZ",67173327600] index-fb5980fc.js:474:1186
16:32:13.016 	 {"applyExtrinsic":2} : utility.BatchInterrupted [0,{"token":"FundsUnavailable"}] index-fb5980fc.js:474:1186
16:32:13.017 	 {"applyExtrinsic":2} : multisig.MultisigExecuted ["5Cqx9dsouTgq48wBuheERrre1fxxdpMktee7RY1T3jZwLNJ4",{"height":5736288,"index":2},"5GH8Zm2Y2gEAQZcAMDc48B6ehxqe4PFFpDTavy51K1wSxbyg","0xbdb093b83c415aed9aa124d3d40402ad95b3c58c453a12b1ef602dfc8f8058a7",{"ok":null}] index-fb5980fc.js:474:1186
16:32:13.017 	 {"applyExtrinsic":2} : balances.Deposit ["5CFuoQ75caszgSZgukcpL6L7Wt4n8hjwtUsB5YvvtrrhhfXf",71587265] index-fb5980fc.js:474:1186
16:32:13.017 	 {"applyExtrinsic":2} : transactionPayment.TransactionFeePaid ["5Cqx9dsouTgq48wBuheERrre1fxxdpMktee7RY1T3jZwLNJ4",71587265,0] index-fb5980fc.js:474:1186
16:32:13.017 	 {"applyExtrinsic":2} : system.ExtrinsicSuccess [{"weight":{"refTime":562650478,"proofSize":6811},"class":"Normal","paysFee":"Yes"}] index-fb5980fc.js:474:1186
16:32:26.625 Transaction status: Finalized index-fb5980fc.js:474:857
16:32:26.625 Finalized block hash 0x597356bcc886bb9cb8d732bd845a49cf342cdce463c4a39e8d67319abb27e998 index-fb5980fc.js:474:2170
```

File: packages/ui/src/components/Toasts/ToastBar.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const ToastBar = ({ toast, className }: Props) => {
       className={className}
       open={true}
       anchorOrigin={{ vertical: VERTICAL_POSITION, horizontal: HORIZONTAL_POSITION }}
-      autoHideDuration={duration || DEFAULT_AUTO_HIDE_DURATION}
+      autoHideDuration={toast.type === 'error' ? 0 : duration || DEFAULT_AUTO_HIDE_DURATION}
```
Comment (Tbaut) [inline packages/ui/src/components/Toasts/ToastBar.tsx:37]:
```suggestion
      autoHideDuration={toast.type === 'error' ? null : duration || DEFAULT_AUTO_HIDE_DURATION}
```

**Review by asnaith — APPROVED** [review]
The latest changes resolve this PR. I can now see 

🟢 Errors in the UI for invalid batch tx failures
🟢 Errors in the UI for an invalid tx that proposed from the multisig

Manual extrinsic errors to be addressed separately in #198

**Pull request merged**